pkg/domain/app_website.go

```diff
@@ -12,11 +13,19 @@ import (
 	"github.com/traPtitech/neoshowcase/pkg/util/ds"
 )
 
+var urlRegexp = regexp.MustCompile(`^[a-z0-9-_]+(\.[a-z0-9-_]+)*$`)
```

Comment:
URLではないかも
（本当は"ドメイン"でもなくて"ホスト"なんだけど、まあよく相互的に使われるからそれはあえて許容してる）

```suggestion
var domainRegexp = regexp.MustCompile(`^[a-z0-9-_]+(\.[a-z0-9-_]+)*$`)
```